src/modules/AdminDashboard/components/tabs/Movie/index.tsx

```diff
@@ -0,0 +1,147 @@
+import React, { useEffect, useState } from "react";
+import { Button, Tabs } from "antd";
+import { useGetMoviesQuery } from "../../../../../app/api/moviApi";
+import { Avatar, List, message } from "antd";
+import VirtualList from "rc-virtual-list";
+import { Movie } from "../../../../../app/api/types";
+import Create from "./Components/Create";
+import Edit from "./Components/Edit";
+import Item from "antd/es/list/Item";
+import {Modal } from 'antd';
+function MovieInfo() {
+  const [isShowMode, setIsShowMode] = useState<boolean>(false);
+  const [isCreateMode, setIsCreateMode] = useState<boolean>(false);
+  const [editId, setEditId] = useState<number>(-1);
+  const {
+    data: movies,
+    error: movieError,
+    isLoading: movieLoading,
+  } = useGetMoviesQuery();
+  const [forceRender, setForceRender] = useState(false);
+
+  const [open, setOpen] = useState(false);
+  const [confirmLoading, setConfirmLoading] = useState(false);
+  const [modalText, setModalText] = useState('Content of the modal');
+
+  const showModal = () => {
+    setOpen(true);
+  };
+
+  const handleOk = () => {
+    setModalText('The modal will be closed after two seconds');
+    setConfirmLoading(true);
+    setTimeout(() => {
```

Comment:
Why are we using timeout here. We try to use such timeout or intervals in react code unless needed